h2o-algos/src/test/java/hex/ensemble/StackedEnsembleTest.java

```diff
@@ -1048,56 +1049,6 @@ public void testMissingFoldColumn_blendingFrame() {
     }
   }
 
-  @Test
-  public void testInvalidFoldColumn_trainingFrame() {
```

Comment:
Since SE now overrides the ignore logic so it keeps columns used by base models/metalearner, this test has no function since it assumes that SE will ignore invalid (String) column which is used by a base model.

Similar test that tests situation when we are missing fold column is precedes this one.